pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -549,6 +555,18 @@ const (
 	LazyActivation ActivationPreference = "Lazy"
 )
 
+// OrphaningDeletionPolicyType indicates how removal action of managed resources
+type OrphaningDeletionPolicyType string
+
+const (
+	// OrphaningDeletionNever means that the resource in member clusters will be deleted when removal of managed resources
+	OrphaningDeletionNever OrphaningDeletionPolicyType = "Never"
+
+	// OrphaningDeletionComplete means that the resource in member clusters completely independent of karmada
+	// Not only cleanup the labels/annotations from the resource on member clusters, but also delete work object in control plane
+	OrphaningDeletionComplete OrphaningDeletionPolicyType = "Complete"
```

Comment:
Complete sounds a bit confusing. Is it possible to call it `CompleteDelete`?

Reply:
ArgoCD uses the term ["cascade"](https://argo-cd.readthedocs.io/en/stable/user-guide/app_deletion/)

Reply:
> It seems that using ["cascade"](https://argo-cd.readthedocs.io/en/stable/user-guide/app_deletion/) would be better, as it can reduce the learning cost for users.

+1

It seems that using ["cascade"](https://argo-cd.readthedocs.io/en/stable/user-guide/app_deletion/) would be better, as it can reduce the learning cost for users.